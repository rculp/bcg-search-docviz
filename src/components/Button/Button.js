import { Button } from 'semantic-ui-react';
import './Button.scss';

export default Button;
