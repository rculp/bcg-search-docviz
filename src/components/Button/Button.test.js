import { shallow } from 'enzyme';

import React from 'react';

import Button from './Button';

describe('Button', () => {
  it('renders and matches our snapshot', () => {
    const component = shallow(
      <Button />
    );
    expect(component).toMatchSnapshot();
  });
});