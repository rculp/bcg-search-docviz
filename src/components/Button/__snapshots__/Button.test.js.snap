// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders and matches our snapshot 1`] = `
<button
  className="ui button"
  onClick={[Function]}
  role="button"
/>
`;
